app/plugins/errors.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
           request.log('error', {
             statusCode,
             data: response.data,
-            message: response.message
+            message: response.message,
+            stack: response.stack
```

Comment:
What does this add? Not come across `response.stack` before

Reply:
It adds the stack trace to the error logs.

Currently it just logs the error message.  Which is great, but I want to know which file and line threw the error.

We should probably have this in the other repos too.